src/main/java/org/embulk/input/jira/util/JiraUtil.java

```diff
@@ -0,0 +1,272 @@
+            result = parser.parse(value);
+        }
+        catch (Exception e) {
+        }
+        return result;
+    }
+
+    /*
+     * For getting the Long value of the node
+     * Sometime if error occurs (i.e a JSON value but user modified it as long) then return null
+     * */
+    private static Long getLongValue(JsonElement value)
+    {
+        Long result = null;
+        try {
+            result = value.getAsLong();
+        }
+        catch (Exception e) {
+        }
+        return result;
+    }
+
+    /*
+     * For getting the Double value of the node
+     * Sometime if error occurs (i.e a JSON value but user modified it as double) then return null
+     * */
+    private static Double getDoubleValue(JsonElement value)
+    {
+        Double result = null;
+        try {
+            result = value.getAsDouble();
+        }
+        catch (Exception e) {
+        }
+        return result;
+    }
+
+    /*
+     * For getting the Boolean value of the node
+     * Sometime if error occurs (i.e a JSON value but user modified it as boolean) then return null
+     * */
+    private static Boolean getBooleanValue(JsonElement value)
+    {
+        Boolean result = null;
+        try {
+            result = value.getAsBoolean();
+        }
+        catch (Exception e) {
+        }
+        return result;
+    }
+
+    public static void addRecord(Issue issue, Schema schema, PluginTask task, PageBuilder pageBuilder)
+    {
+        schema.visitColumns(new ColumnVisitor() {
+            @Override
+            public void jsonColumn(Column column)
+            {
+                JsonElement data = issue.fetchValue(column.getName());
+                if (data.isJsonNull() || data.isJsonPrimitive()) {
+                    pageBuilder.setNull(column);
+                }
+                else {
+                    pageBuilder.setJson(column, new JsonParser().parse(data.toString()));
+                }
+            }
+
+            @Override
+            public void stringColumn(Column column)
+            {
+                JsonElement data = issue.fetchValue(column.getName());
+                if (data.isJsonNull()) {
+                    pageBuilder.setNull(column);
+                }
+                else if (data.isJsonPrimitive()) {
+                    pageBuilder.setString(column, data.getAsString());
+                }
+                else if (data.isJsonArray()) {
+                    pageBuilder.setString(column, String.join(",", StreamSupport.stream(data.getAsJsonArray().spliterator(), false)
+                            .map(obj -> {
+                                if (obj.isJsonPrimitive()) {
+                                    return obj.getAsString();
+                                }
+                                else {
+                                    return obj.toString();
+                                }
+                            })
+                            .collect(Collectors.toList())));
```

Comment:
Code style: could be simplified with `Collectors#joining`

Reply:
Collectors#joining is for string.
When I changed the test failed. Hence I would like to keep it as current

Reply:
This failed?
```suggestion
pageBuilder.setString(column,
        StreamSupport.stream(data.getAsJsonArray().spliterator(), false)
                .map(obj -> {
                    if (obj.isJsonPrimitive()) {
                        return obj.getAsString();
                    }
                    else {
                        return obj.toString();
                    }
                })
                .collect(Collectors.joining(",")));
```

Reply:
Done